### PR TITLE
2 typo fixed

### PR DESCRIPTION
**Pull request by RealFakeAccount**
These are the typos in Module 5 answer. Please check it.

image

### CODE REVIEW

**Comment by RealFakeAccount** [comment]
I just found that this is the 200th pull requests!

File: book/modules/module5-exercises.tex [hunk 1]
```diff
@@ -208,7 +208,7 @@
 					\[
 						\Comp_{\vec v}(\vec a) = \frac{\vec
 						a \cdot \vec v}{\vec v \cdot
-						\vec v}\vec u = \frac{2}{2}\vec
+						\vec v}\vec v = \frac{2}{2}\vec
 						v = \mat{0\\1\\1}.
```
Comment (siefkenj) [inline book/modules/module5-exercises.tex:212]: Shouldn't this vector be in R^2?

Comment (RealFakeAccount) [reply book/modules/module5-exercises.tex:212]: image
Sorry about that. This is the typo in the answer of Question 7

**Comment by siefkenj** [comment]
Thanks!

**Pull request merged**